features/checkout/addressing_order/filling_shipping_and_billing_address_details.feature

```diff
@@ -7,14 +7,16 @@ Feature: Addressing an order
     Background:
         Given the store operates on a single channel in "France"
         And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store has "Raven Post" shipping method with "€10.00" fee
         And I am logged in customer
 
     @ui
     Scenario: Address an order without different billing address
         Given I have product "PHP T-Shirt" in the cart
         And I am at the checkout addressing step
         When I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
-        And I proceed to the next step
+        And I proceed to the shipping step
```

Comment:
As we should somehow differentiate "next step" action between steps (for now its addressing and shipping)... Or maybe we could use ``CurrentPageResolver`` for that? which option is better in this case?